.buildkite/pipeline.trigger.integration.tests.sh

```diff
@@ -49,6 +49,22 @@ for test in ${CHECK_PACKAGES_TESTS[@]}; do
     fi
 done
 
+pushd test/packages/false_positives > /dev/null
+for package in $(find . -maxdepth 1 -mindepth 1 -type d) ; do
+    package_name=$(basename ${package})
+    echo "      - label: \":go: Running integration test: ${package_name}\""
+    echo "        key: \"integration-false_positives-${package_name}\""
+    echo "        command: ./.buildkite/scripts/integration_tests.sh -t test-check-packages-false-positives -p ${package_name}"
+    echo "        agents:"
+    echo "          provider: \"gcp\""
+    echo "        artifact_paths:"
+    echo "          - build/test-results/*.xml"
+    echo "          - build/elastic-stack-dump/check-*/logs/*.log"
+    echo "          - build/elastic-stack-dump/check-*/logs/fleet-server-internal/**/*"
```

Comment:
Should we do as in `parallel` packages and upload logs to the internal bucket ? cc @jsoriano

Reply:
Sorry, I missed this comment.

I would do the same as in `parallel`, so we don't forget to do it here if we want to avoid risks. I guess that in `parallel` we are doing this to avoid leaking secrets related to cloud integrations or the terraform test runner.

Reply:
@jsoriano Does this add any value? I know why we did it in `parallel` , as we used real cloud instances and secrets. These are mere mocked tests , or?

Reply:
Tests packages including terraform scenarios are actually executed.

Reply:
Done.. Pushed